frontend/app/src/hooks/usePermission.ts

```diff
@@ -3,6 +3,7 @@ import { DateTimeParam, useQueryParam } from "use-query-params";
 import { useAuth } from "./useAuth";
 
 interface UsePermission {
+  isAdmin: boolean;
   write: {
     allow: boolean;
     message: string | null;
```

Comment:
how about keeping the same structure than write/read? I'll enable us to specify a message

Reply:
yes why not